libs/shared/ui-components/src/Footer/FooterColumnProvider/FooterNavigation.tsx

```diff
@@ -12,14 +12,18 @@ export const FooterNavigation: FC<TFooterNavigationProps> = ({
     <h2 className="pb-[1.3rem] mb-4 text-[1.6rem] font-bold leading-[3rem] border-b-[0.5px] border-black xl:border-white">
       {title}
     </h2>
-    <nav className="flex flex-col">
-      {links.map(({ linkText, linkUrl, _uid }) => (
-        <Link href={`/${linkUrl.cached_url}`} key={_uid}>
-          <a className="text-[1.4rem] font-normal leading-[3.3rem]">
-            {linkText}
-          </a>
-        </Link>
-      ))}
-    </nav>
+    <div className="flex flex-col">
```

Comment:
Per guidance from WebAIM, I decided that [none of the links in the footer should be enclosed in a `nav` tag](https://webaim.org/techniques/semanticstructure/).

I wasn't sure whether they should be wrapped in a list (still not sure), but I just copied a11yproject.com on this one.